### PR TITLE
Add issues related to failing bags

### LINKED ISSUE #995: Problem: encoding requirements for bags are not represented in documentation
**Version of the documentation**
1.10 (only happens with bagit-python, did not happen with bagit-java, which was replaced in #246)

**Page (and section, if applicable) where the issue occurs**
https://www.archivematica.org/en/docs/archivematica-1.10/user-manual/transfer/bags/

**Description of the issue**
As described by @replaceafill [here](https://github.com/archivematica/Issues/issues/977#issuecomment-551311895), if you have inconsistent encoding info in the bag, the bag will fail verification.

**Suggested fix**
Add a section about bag requirements (encoding, structure, etc.)

Related to #977 and #694 

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] Details about this issue have been added to the release notes (if applicable)

### PR DESCRIPTION
**Pull request by ablwr**
Resolves https://github.com/archivematica/Issues/issues/995

@sallain Are there other common issues associated with failing bags that we can add here? Did you envision something different for the above issue?

### CODE REVIEW

**Comment by sallain** [comment]
@ablwr I think this is just what I was envisioning. I don't know if it's a common issue, but might be worth adding "editing a file that has already been bagged" to the list of fail conditions? (Confession: I've definitely done this while testing bag stuff, even though I know better :joy:)

File: user-manual/transfer/bags.rst [hunk 1]
```diff
@@ -10,11 +10,26 @@ unzipped bags by using the appropriate :ref:`transfer type <transfer-types>`.
...
 * :ref:`Adding metadata to bags <adding-metadata-bags>`
 * :ref:`Index and search bag metadata <searching-bags>`
 
+.. _bag-structure-requirements:
+
+Bag structure requirements
+--------------------------
+
+Bags must adhere to requirements of the `BagIt`_ specification. If not,
+Archivematica will error and files will be cleaned up and removed.
```
Comment (sallain) [inline user-manual/transfer/bags.rst]:
```suggestion
the transfer will fail and files will be cleaned up and removed.
```

**Review by sallain — APPROVED** [review]
One small rec for clarity but otherwise LGTM!

**Comment by ablwr** [comment]
@sallain yes thanks! It was definitely at the end of the day for my brain yesterday. I also added your recommendation of file editing (I'm guilty too) and referenced structural issues and how to find out more.

**Review by sallain — APPROVED** [review]

**Pull request merged**